code/modules/clothing/suits/chaplainsuits.dm

```diff
@@ -85,3 +87,129 @@
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS
 	flags_inv = HIDEJUMPSUIT
 	allowed = list(/obj/item/megaphone/clown, /obj/item/soap, /obj/item/food/pie/cream, /obj/item/bikehorn, /obj/item/bikehorn/golden, /obj/item/bikehorn/airhorn, /obj/item/instrument/bikehorn, /obj/item/reagent_containers/food/drinks/soda_cans/canned_laughter, /obj/item/toy/crayon, /obj/item/toy/crayon/spraycan, /obj/item/toy/crayon/spraycan/lubecan, /obj/item/grown/bananapeel, /obj/item/food/grown/banana)
+
+/obj/item/clothing/head/helmet/chaplain/clock
+	name = "forgotten helmet"
+	desc = "It has the unyielding gaze of a god eternally forgotten."
+	icon_state = "clockwork_helmet"
+	inhand_icon_state = "clockwork_helmet_inhand"
+	armor = list(MELEE = 50, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 0, BIO = 0, FIRE = 80, ACID = 80)
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
+	strip_delay = 8 SECONDS
+	dog_fashion = null
+
+/obj/item/clothing/suit/chaplainsuit/clock
+	name = "forgotten armour"
+	desc = "It sounds like hissing steam, ticking cogs, gone silent, It looks like a dead machine, trying to tick with life."
+	icon_state = "clockwork_cuirass"
+	inhand_icon_state = "clockwork_cuirass_inhand"
+	allowed = list(/obj/item/storage/book/bible, /obj/item/nullrod, /obj/item/reagent_containers/food/drinks/bottle/holywater, /obj/item/storage/fancy/candle_box, /obj/item/candle, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
+	slowdown = 0
+	clothing_flags = NONE
+
+/obj/item/clothing/head/helmet/chaplain
+	name = "crusader helmet"
+	desc = "Deus Vult."
+	icon_state = "knight_templar"
+	inhand_icon_state = "knight_templar"
+	armor = list(MELEE = 50, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 0, BIO = 0, FIRE = 80, ACID = 80)
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
+	strip_delay = 80
+	dog_fashion = null
+
+/obj/item/clothing/suit/chaplainsuit/templar
```

Comment:
you made these no longer subtypes of riot armor. they won't have any armor now

Reply:
gave the skele a riot subtype version back

Reply:
wdym its for the normal chaplain

Reply:
wdym its for normal chaplain

Reply:
huh armor for a chaplain? shouldnt it just be cosmetic?